numba_dpex/tests/debugging/test_info.py

```diff
@@ -85,6 +88,34 @@ def test_info_args(
     for arg in expected_args:
         app.child.expect(arg)
```

Comment:
Why was this test cutted?

Reply:
I have split this into two tests, one tests for `info_args` and other does for `expected_info` as they are two separate things.

Reply:
This should be mentioned in the description